apps/server/src/modules/user/legacy/controller/dto/consent.response.ts

```diff
@@ -0,0 +1,18 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { UserConsentResponse } from './user-consent.response';
+import { ParentConsentResponse } from './parent-consent.response';
+
+export class ConsentResponse {
```

Comment:
There is a problem with the OpenAPI spec, because a class of name "ConsentResponse" already exists in our code: https://github.com/hpi-schul-cloud/schulcloud-server/blob/main/apps/server/src/modules/oauth-provider/controller/dto/response/consent.response.ts

This is a general problem with duplicate class names, see: https://github.com/nestjs/nest/issues/2946

So we need another name for this class here.